llvm/utils/lit/tests/Inputs/shtest-shell/curly-brace.txt

```diff
@@ -0,0 +1,19 @@
+## Test curly braces used in grouping commands.
+
+## Test one command inside curly brace.
+# RUN: { echo bar; } | FileCheck --check-prefix=ONE-CMD %s
+
+# ONE-CMD: bar
+
+## Test two commands inside curly brace.
+# RUN: { echo foo; echo bar; } | FileCheck --check-prefix=TWO-CMDS %s
+
+# TWO-CMDS: foo
+# TWO-CMDS: bar
+
+## Test nested curly brace.
+# RUN: { echo foo > %t/temp.txt; { echo bar; echo baz; }; cat %t/temp.txt; } | FileCheck --check-prefix=NESTED %s
```

Comment:
I don't see a `mkdir %t`, so I'm not sure this works as intended. If you don't need other files, you can just use `%t`.